examples/deployment/scripts/transfer/gold-to-john.js

```diff
@@ -0,0 +1,21 @@
+const algosdk = require('algosdk');
+const {
+  printCreatedAsset,
+  printAssetHolding,
+  transferMicroAlgos,
+  asaOptIn,
+  transferAsset
+} = require('../../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
```

Comment:
let's add a comment above the script, explaining that each execution through `algob run` will create and execute transfer transaction....

Reply:
I'll add this:
```
// Create "gold" Algorand Standard Asset (ASA)
// Accounts are loaded from config
// To use ASA accounts have to opt-in and owner is opt-in by default
// john-account is transferred some funds to execute opt-in transaction
```